finesse/hardware/plugins/em27/dummy_opus_interface.py

```diff
@@ -65,6 +65,10 @@ def __init__(
     ) -> None:
         """Create a new OPUSStateMachine.
 
+        The state diagram looks like this:
+
+        ![](../../../../../OPUSStateMachine.png)
```

Comment:
These sort of 5-levels-deep relative imports look awful. I was thinking that maybe we could use some sort of macro for the base path of the documentation, but apparently that is not possible :( 

https://github.com/fralau/mkdocs_macros_plugin/discussions/174